views/tx.tmpl

```diff
@@ -248,6 +248,16 @@
                 <td class="text-right medium-sans text-nowrap pr-2 py-2">Rate:</td>
                 <td class="text-left py-1 text-secondary">{{.FeeRate}}/kB</td>
               </tr>
+              {{if $.SwapsFound}}
+                <tr>
+                  <td class="text-right medium-sans text-nowrap pr-2 py-2"
+                    >Swaps Info:
+                  </td>
+                  <td class="text-left py-1 text-secondary"><a
+                    href="/api/tx/swaps/{{.TxID}}?indent=true" data-turbolinks="false">swaps</a>
+                  </td>
+                </tr>
+              {{end}}
```

Comment:
Since we are no longer able to show details of swaps related to a tx, I added this link to the api endpoint that can display that info.